docs/sql-migration-guide.md

```diff
@@ -22,6 +22,10 @@ license: |
 * Table of contents
 {:toc}
 
+## Upgrading from Spark SQL 3.4 to 3.5
+
+- Since Spark 3.5, the JDBC options related to DS V2 pushdown are `true` by default. e.g. `pushDownAggregate`. To restore the legacy behavior, please set them to `false`. e.g. set `spark.sql.catalog.your_catalog_name.pushDownAggregate` to `false`.
```

Comment:
- `default. e.g.` -> `default, e.g.`
- We had better enumerate all changes for the end users. If we mentioned only one configuration, they have no idea how to recover the others.

Reply:
It hears good to me.